CONTRIBUTING.md

```diff
@@ -243,18 +243,87 @@ If in doubt, you can always ask for guidance in the Pull Request or on
 [IRC in the #node-dev channel](https://webchat.freenode.net?channels=node-dev&uio=d4).
 
 Feel free to post a comment in the Pull Request to ping reviewers if you are
-awaiting an answer on something.
+awaiting an answer on something. If you encounter words or acronyms that
+seem unfamiliar, check out this
+[glossary](https://sites.google.com/a/chromium.org/dev/glossary).
 
+Note that multiple commits often get squashed when they are landed (see the
+notes about [commit squashing](#commit-squashing)).
 
 ### Step 8: Landing
 
-Once your Pull Request has been reviewed and approved by at least one Node.js
-Collaborators (often by saying LGTM, or Looks Good To Me), and as long as
-there is consensus (no objections from a Collaborator), a
-Collaborator can merge the Pull Request . GitHub often shows the Pull Request as
- `Closed` at this point, but don't worry. If you look at the branch you raised
- your Pull Request against (probably `master`), you should see a commit with
- your name on it. Congratulations and thanks for your contribution!
+In order to get landed, a Pull Request needs to be reviewed and
+[approved](#getting-approvals-for-your-pull-request) by
+at least one Node.js Collaborator. If the Pull Request touches
+more than the documentation, then it also need to pass a
```

Comment:
Well, what is currently documented is that you should always run `make -j4 test` no matter what, which is strictly correct. What isn't documented is that in practice you can sometimes get away with just running the linter (or the specific test you changed). That's not something we really want to encourage, especially as it's a tricky issue (some doc changes do need `make test`, some test changes do need `make test`). Documenting something is effectively legitimising it, which I (personally) don't think we want to do.